records_versioning/migrations/0001_initial.py

```
@@ -19,44 +20,18 @@ class Migration(migrations.Migration):
-                    models.Index(
-                        fields=['resource_type', 'resource_id', '-version'],
-                        name='recordver_type_id_ver_desc_idx',
-                    ),
-                ],
+                'unique_together': {('resource_type', 'resource_id', 'version')},
+                'indexes': ["models.Index(fields=['resource_type', 'resource_id', 'version'], name='records_ver_resourc_8f9a0b_idx')", "models.Index(fields=['changed_at'], name='records_ver_changed_2c3d4e_idx')"],
             },
         ),
-    ]
\ No newline at end of file
+    ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)